access-mgmt/README.adoc

```diff
@@ -0,0 +1,469 @@
+* A single operation (such as `Write`) for a resource or all operations for a resource
+
+You can also allow other users or service accounts to manage access to the resources in your instance for you.
+
+If two or more permission settings in an ACL match a request being made to the Kafka broker and at least one of the matching permissions specifies that the action is denied, then the request is denied.
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://kafka.apache.org/documentation/#security_authz[Authorization and ACLs^] in Kafka documentation
+
+=== Supported ACL permissions in {product}
+
+An ACL acts as a mapping of permitted operations on specified resources for a selected account or for all accounts in an organization. An account can be a user account or a service account. Operations correspond to Kafka APIs or request types that relate to the specified resource.
+
+For example, a `Read` operation for a `Topic` resource corresponds to the `Fetch`, `OffsetCommit`, and `TxnOffsetCommit` Kafka requests. A `Write` operation for a `Topic` resource corresponds to the `Produce` and `AddPartitionsToTxn` Kafka requests.
+
+The following table lists the supported ACL permissions in {product}.
+
+NOTE: The resource identifier `Is` supports the wildcard character `*` to denote any occurrences of the specified resource. For example, `Topic` = `is *` means any topic in a Kafka instance.
+
+.Supported ACL permissions
+[cols="25%,25%,25%,25%"]
+|===
+h|Resource type
+h|Resource identifier
+h|Access type
+h|Operations
+
+|`Consumer group`
+
+(For consumer group access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Read`
+* `Delete`
+* `Describe`
+
+|`Topic`
+
+(For access to a topic)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Read`
+* `Write`
+* `Create`
+* `Delete`
+* `Alter`
+* `Alter configs`
+* `Describe`
+* `Describe configs`
+
+|`Transactional ID`
+
+(For producer access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Write`
+* `Describe`
+
+|`Kafka instance`
+
+(For access to Kafka instance permissions in ACLs)
+|None
+a|
+* `Allow`
+* `Deny`
+a|
+* `Alter`
+* `Describe`
+|===
+
+[role="_additional-resources"]
+.Additional resources
+https://kafka.apache.org/documentation/#security_authz_primitives[Authorization Primitives^] in Kafka documentation
+
+[id="proc-setting-permissions_{context}"]
+== Setting account permissions in a Kafka instance in {product}
+
+[role="_abstract"]
+In {product}, you can create Access Control Lists (ACLs) in your Kafka instances and set permissions for how other user accounts or service accounts can interact with an instance and its resources. You can manage access for only the Kafka instances that you create or for the instances that the owner has enabled you to access and alter.
```

Comment:
For searchability, could we reverse a bit to "Create Access Control Lists (ACLs) in your OpenShift for Apache Kafka instances to set permissions..." 
or
"Use Access Control Lists (ACLs) to manage access to Kafka instances and resources in [long honkin' product name]"

Reply:
I actually thought that providing the product as context was best for searchability, plus it makes it less clunky, and plus we seem to have a pattern of that around. As I try some of these other alternatives, it just doesn't flow as well to my ear and removes the important context for when users hit it from a search. Not sure how to alter it and still achieve that, and still not have it clunky.

Reply:
Ok -- if you have the chance, maybe check with Ben? If not, we can revisit later if needed. :)

Reply:
@bhardesty Would you mind weighing in on this question? ^^ Would love your perspective.

Reply:
I like the original title the best. Setting permissions is the goal, ACLs are an implementation detail. Also, the intro para explains the use of ACLs, so we don't completely lose searchability.

Reply:
Regarding the intro paragraph, I think it makes sense to first convey the context, and then describe what you can do within that context. So I think I like the original formulation the best, unless it proves to create significant searchability issues.